src/MemoryOptions.php

```diff
@@ -16,98 +16,102 @@
  */
 class MemoryOptions extends AdapterOptions
 {
-    /**
```

Comment:
If you didn't recormat this file on purpose,I suggest:

 * changing just what you intended
 * reverting all the rest
 * squashing
 * force pushing